pkg/minikube/cluster/commands.go

```diff
@@ -16,10 +16,33 @@ limitations under the License.
 
 package cluster
 
-var startCommand = `
-# Run with nohup so it stays up. Redirect logs to useful places.
-PATH=/usr/local/sbin:$PATH nohup sudo /usr/local/bin/localkube start --generate-certs=false > /var/log/localkube.out 2> /var/log/localkube.err < /dev/null &
-`
+import (
+	gflag "flag"
+	"fmt"
+
+	"k8s.io/minikube/pkg/minikube/constants"
+)
+
+const (
+	remoteLocalKubeErrPath = "/var/log/localkube.err"
+	remoteLocalKubeOutPath = "/var/log/localkube.out"
+)
 
 // Kill any running instances.
 var stopCommand = "killall localkube | true"
+
+func GetStartCommand() string {
+	startCommandFormattable := `
+# Run with nohup so it stays up. Redirect logs to useful places.
+PATH=/usr/local/sbin:$PATH nohup sudo /usr/local/bin/localkube start %s--generate-certs=false > %s 2> %s < /dev/null &
+`
+	concatFlags := ""
+	for _, logFlag := range constants.LogFlags {
+		if logVal := gflag.Lookup(logFlag); logVal != nil {
+			if logVal.Value.String() != "" {
```

Comment:
Nit:
You could do something like this:
flagVals := []string
if val := logVal.Value.String(); val != "" {
  flagVals = append(flagVals, fmt.Springtf("--%s %s", logFlag, val))

flags := strings.Join(flagVals, " ")